.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
What do you think about naming this file `launch.json.template` (Or something similar?) because this file can be updated but it should not be committed.

Reply:
I think this seems reasonable. There is a lot of conflicting recommendation on whether it is worth checking in `launch.json`. Do you think it makes sense to keep the `tasks.json`? or should that also be a template?

Reply:
To be consistent, `tasks.json` could be a template but I do not have a strong opinion on it.